macros/snowplow_media_player_base_events_this_run/source_url_col.sql

```diff
@@ -1,4 +1,7 @@
--- source_url column in the snowplow_media_player_base_events_this_run table
+{#
+  Returns the value source_url column in the snowplow_media_player_base_events_this_run table
+  based on the columns for the url in YouTube context and current_src in media context.
+#}
```

Comment:
Sorry, the macros should be documented in a similar way to the models/columns, see this example here https://github.com/snowplow/dbt-snowplow-utils/blob/main/macros/utils/schema.yml